app/Config/Cache.php

```diff
@@ -110,11 +116,11 @@ class Cache extends BaseConfig
 	|
 	*/
 	public $validHandlers = [
-		'dummy'     => \CodeIgniter\Cache\Handlers\DummyHandler::class,
-		'file'      => \CodeIgniter\Cache\Handlers\FileHandler::class,
-		'memcached' => \CodeIgniter\Cache\Handlers\MemcachedHandler::class,
-		'predis'    => \CodeIgniter\Cache\Handlers\PredisHandler::class,
-		'redis'     => \CodeIgniter\Cache\Handlers\RedisHandler::class,
-		'wincache'  => \CodeIgniter\Cache\Handlers\WincacheHandler::class,
+		'dummy'     => DummyHandler::class,
```

Comment:
Because these are configuration values that end users are expected to be able to easily change, I prefer these to show the full namespace. Please revert. Also the other similar changes in this PR should be reverted also.